tests/src/core/testqgsrasterlayer.cpp

```diff
@@ -112,6 +114,8 @@ class TestQgsRasterLayer : public QObject
     QgsRasterLayer *mPngRasterLayer = nullptr;
     QgsRasterLayer *mGeoJp2RasterLayer = nullptr;
 
+    QgsRasterLayerTemporalProperties *temporalProperties = nullptr;
```

Comment:
Try to avoid members like this in tests, and instead try to keep each individual test self-contained. It avoids a massive overflow of failing tests when one test which modifies the member variable fails.